rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -7890,7 +7890,7 @@ void opcode_02e6(const ScriptArguments& args, const ScriptObject object, const S
     auto cutscene = args.getObject<CutsceneObject>(0);
     std::string animName = arg2;
     std::transform(animName.begin(), animName.end(), animName.begin(), ::tolower);
-    auto anim = args.getWorld()->data->animations[animName];
+    auto anim = args.getWorld()->data->animationsCutscene[animName];
```

Comment:
An `AnimationSet` is a `std::map`. Using the `operator[]` will create a new entry in the map.
So use the `find` member function.